Frameworks/UIKit/UILabel.mm

```diff
@@ -293,6 +200,7 @@ - (void)_initUILabel {
     }
 
     _textBlock = textBlock;
+    [_textBlock setLineStackingStrategy:WXLineStackingStrategyBlockLineHeight];
```

Comment:
maybe in a subsequent pass?

---
In reply to: [105784192](https://github.com/Microsoft/WinObjC/pull/2178#discussion_r105784192) [](ancestors = 105784192,105782303,105781932)